src/admin/localService/loadFile.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+const fs = require("fs");
+const pathJoin = require("path").join;
+
+const readdir = fs.promises.readdir;
+
+/**
+ * load file into server memory
+ */
+async function loadFile(credDir: string, fileName: string): Promise<string> {
+  const filePath = await getFilePath(`${credDir}/`, fileName);
+  if (!filePath)
+    return Promise.reject(
+      new Error(`${fileName} not found. Please enter a full cred repo`)
```

Comment:
Might be good to have a check / error message for having the wrong cred directory vs a file not existing. Also, "Please enter a full cred repo" is a bit confusing, what does "full" mean in this context? Might be good to reword it.

Reply:
I added some usage info to the server logic so users know what repo they're supposed to be using, and how to enter it. If a file doesn't exist in this case, it's pretty safe to assume that they're looking in the wrong directory from my observation, unless they've just never run any cred calculations before. not sure how we'd want to handle that caveat though, since I don't know what the instructions would be, or how to differentiate between a repo that has just never run calculations vs an incorrect repo.